pkg/apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -350,6 +354,12 @@ type SaveAsTemplateOutput struct {
 	ClusterTemplateRevisionName string `json:"clusterTemplateRevisionName,omitempty"`
 }
 
+type AKSStatus struct {
+	UpstreamSpec          *aksv1.AKSClusterConfigSpec `json:"upstreamSpec"`
+	VirtualNetwork        string                      `json:"virtualNetwork"`
+	Subnets               []string                    `json:"subnets"`
```

Comment:
Just want to make sure that these are needed for AKS.

Reply:
It was removed